mifosng-android/src/main/java/com/mifos/mifosxdroid/login/LoginActivity.java

```diff
@@ -210,27 +227,34 @@ public void showProgressbar(boolean show) {
      */
     private void promptUserToByPassTheSSLHandshake() {
         new AlertDialog.Builder(this)
-                .setTitle("SSL Certificate Problem")
-                .setMessage("There is a problem with your SSLCertificate, would you like to " +
-                        "continue? This connection would be unsafe.")
-                .setIcon(android.R.drawable.stat_sys_warning)
-                .setPositiveButton("Continue", new DialogInterface.OnClickListener() {
+                .setTitle(getString(R.string.ssl_certificate_dialog))
```

Comment:
Looks good, Please remove default ```AlertDialog``` with this one https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MaterialDialog.java